### PR TITLE
feat(ctb): Introduce `Semver` on `FaultDisputeGame` + `DisputeGameFactory`

### PR DESCRIPTION
**Pull request by clabby**
## Overview

Removes the hardcoded `VERSION` constant from the dispute contracts in favor of `Semver`, which is the standard in the rest of the contracts codebase.

### CODE REVIEW

File: op-bindings/bindings/disputegamefactory.go
Comment (maurelian) [inline op-bindings/bindings/disputegamefactory.go:1]: Should there be a binding for the `FaultDisputeGame` itself?

**Review by maurelian — APPROVED** [review]
LGTM pending resolution of one comment.

File: op-bindings/bindings/disputegamefactory.go
Comment (clabby) [reply op-bindings/bindings/disputegamefactory.go:1]: Yes there should - we haven't added it just yet. Can do in this PR or a separate one.

Comment (maurelian) [reply op-bindings/bindings/disputegamefactory.go:1]: Separate is fine.

**Comment by clabby** [comment]
@mergifyio queue

**Pull request merged**